internal/helper/podrunner.go

```diff
@@ -0,0 +1,64 @@
+package helper
+
+import (
+	"bytes"
+	"io"
+	"net/url"
+
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+)
+
+func ExecInMCPod(podName string, namespace string, containerName string, command []string) (string, string, error) {
+
+	logger.Log("Running %v in container %q in pod %q", command, containerName, podName)
+
+	tty := true
+
+	restCfg, err := controllerruntime.GetConfig()
+	if err != nil {
+		return "", "", microerror.Mask(err)
+	}
+	coreClient, err := kubernetes.NewForConfig(restCfg)
+	if err != nil {
+		return "", "", microerror.Mask(err)
+	}
```

Comment:
We already have Kubernetes clients for both the MC and the WC that should be used rather than creating new clients:
E.g.
```
state.GetFramework().MC()
```